src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -86,26 +86,39 @@
       <NegotiateTestUpn>testuser@DOMAIN.CONTOSO.COM</NegotiateTestUpn>
     </PropertyGroup>
 
+    <!-- Convention for TestProperties values to use by [ConditionalFact]:
+      'true' = yes, the condition is satisfied
+      'false' = no, the condition is not satisfied
+      empty string = self-detect at runtime if possible, else assume 'false'
+    -->
     <PropertyGroup Condition="'$(Domain_Joined)' == ''">
-      <Domain_Joined>false</Domain_Joined>
+      <Domain_Joined></Domain_Joined>
     </PropertyGroup>
   
     <PropertyGroup Condition="'$(Root_Certificate_Installed)' == ''">
-      <Root_Certificate_Installed>false</Root_Certificate_Installed>
+      <Root_Certificate_Installed></Root_Certificate_Installed>
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Client_Certificate_Installed)' == ''">
-      <Client_Certificate_Installed>false</Client_Certificate_Installed>
+      <Client_Certificate_Installed></Client_Certificate_Installed>
     </PropertyGroup>
   
     <PropertyGroup Condition="'$(SPN_Available)' == ''">
-      <SPN_Available>false</SPN_Available>
+      <SPN_Available></SPN_Available>
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Kerberos_Available)' == ''">
```

Comment:
Honestly, I was waiting for requested feedback on https://github.com/dotnet/wcf/issues/958 to settle on the conditions.  Example, when do we need to distinguish between root and client cert (do you always need to require root if you are testing client certs)?  When do we need to distinguish between client and server?  Do we need to distinguish between domain-joined server but not a domain-joined client.

This PR was a way to say "here they are -- tell me what I'm missing".